app/i18n/en/index.ts

```diff
@@ -2336,6 +2336,14 @@ const en: BaseTranslation = {
       Payments: {
         title: "Payments",
         description: "Notifications related to sending and receiving payments.",
+      },
+      Marketing: {
+        title: "Promotions and updates",
```

Comment:
not a big fan of `Promotions`; for me this is a trigger word that if I see that, I'll turn it off. I prefer `new features and general updates` of something like that

Reply:
Changed it